var/spack/repos/builtin/packages/qt/package.py

```diff
@@ -51,7 +51,7 @@ class Qt(Package):
             description="Build the Webkit extension")
     variant('examples',   default=False,
             description="Build examples.")
-    variant('framework',   default=False,
+    variant('framework',   default=(sys.platform == 'darwin'),
```

Comment:
In the `qt` package, since there are many configuration differences when using Mac, there's a constant `MACOS_VERSION` whose value is the version tuple if a mac OS system, otherwise `None`. To be consistent with the other dependencies and variants listed in the QT package, this should be changed to `default=(MACOS_VERSION is not None)` or `default=bool(MACOS_VERSION)`.

Reply:
done.